src/main/resources/translator.properties

```diff
@@ -762,8 +786,12 @@ hudson.plugins.parameterizedtrigger.BuildTrigger.configs.hudson.plugins.paramete
 hudson.plugins.parameterizedtrigger.BuildTriggerConfig.condition = condition
 hudson.plugins.parameterizedtrigger.BuildTriggerConfig.projects = projects
 
-hudson.plugins.parameterizedtrigger.BlockableBuildTriggerConfig.configs = parameters
-hudson.plugins.parameterizedtrigger.BlockableBuildTriggerConfig.configs.type = OBJECT
+#hudson.plugins.parameterizedtrigger.BlockableBuildTriggerConfig.configs = parameters
+#hudson.plugins.parameterizedtrigger.BlockableBuildTriggerConfig.configs.type = OBJECT
```

Comment:
Should we delete these commented out lines?

Reply:
Just did!